web/server/web_client.c

```diff
@@ -847,6 +857,16 @@ static inline HTTP_VALIDATION http_request_validate(struct web_client *w) {
         w->mode = WEB_CLIENT_MODE_OPTIONS;
     }
     else if(!strncmp(s, "STREAM ", 7)) {
+#ifdef ENABLE_HTTPS
+        if ( (w->ssl.flags) && (netdata_use_ssl_on_stream & NETDATA_SSL_FORCE)){
+            w->header_parse_tries = 0;
+            w->header_parse_last_size = 0;
+            web_client_disable_wait_receive(w);
+            error("The server is configured to always use encrypt connection, please enable the SSL on slave.");
```

Comment:
Please add the slave's host name here, if you can.

Reply:
It was added with the last pull, thanks!